anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
+"""
+    Script to analyze sbitreadout data
+    By: Caterina Aruta (caterina.aruta@cern.ch) and Francesco Ivone (francesco.ivone@cern.ch)
+    """
+if __name__ == '__main__':
+    import ROOT as r
+    import numpy as np
+    import root_numpy as rp
+    import glob
+    import sys
+    import os
+    import time
+    from optparse import OptionParser
+    from subprocess import call
+    from array import array
+    from gempython.utils.nesteddict import nesteddict as ndict
+    from gempython.gemplotting.mapping.chamberInfo import chamber_iEta2VFATPos
+    from gempython.gemplotting.mapping.chamberInfo import chamber_vfatPos2iEtaiPhi as vfat_to_etaphi
+    from gempython.gemplotting.utils.anautilities import make3x8Canvas, saveSummaryByiEta, getMapping
+    from gempython.gemplotting.utils.anaoptions import parser
+    import pkg_resources
+
+    parser.set_defaults(outfilename="sbitReadOut.root")
+    (options, args) = parser.parse_args()
+    path = options.filename
+    size = ((options.GEBtype).lower())
+
+    if not os.path.isdir(path):
+        raise OSError(2, "No such file or directory", path)
+    else:
+        pass
+
+    if size not in ('long', 'short'):
+        raise AssertionError("Invalid value of GEBtype")
+    else:
+        pass
+
+    filename = options.filename+"/sbitReadOut"
+    outfilename = options.outfilename
+    print("Analyzing: '%s'" % path)
+    os.system("mkdir "+filename)
+
+    """
+    At the moment the output of the sbitReadout.py are .dat files with headers that will automatically fill the ROOT TTree. 
+    However ther is one last ':' in the first line that shouldn't be there; consequently the ReadFile function is not able to understand the header. So before reading these .dat files, one has to be sure to remove the ':'
+    To achieve this the following command is needed (removes the last : in the header of all .dat files in the input path)
+    """
```

Comment:
Lines 45-47 please address @jsturdy's comment [here](
https://github.com/cms-gem-daq-project/vfatqc-python-scripts/pull/208#discussion_r226916707)

Reply:
This can be fixed by editing this loop here:

https://github.com/cms-gem-daq-project/xhal/blob/645b5649c0f87ba3d82258082fbf7a4dff5aa853/xhalcore/src/common/rpc_manager/amc.cc#L113-L117

Add an if-else statement that checks if `cluster == 7`, if so then drop the `:` from the printed text, otherwise (`else`) print what is shown.